device_backends/NumericAddressedBackend/include/NumericAddressedBackend.h

```diff
@@ -174,12 +174,12 @@ namespace ChimeraTK {
      * modified. This map is filled in the constructor. The rest of the code is accessing it through the const
      * _interruptDispatchers reference, which is thread safe.
      */
-    std::map<std::pair<int, int>, boost::shared_ptr<NumericAddressedInterruptDispatcher>> _interruptDispatchersNonConst;
+    std::map<uint32_t, boost::shared_ptr<NumericAddressedInterruptDispatcher>> _primaryInterruptDispatchersNonConst;
```

Comment:
A more elegant (and safer) way would be to store only a const version of the map (not a reference, make the entire map const) and fill it in the constructor initialiser list (via a function call there). This would prevent any modification of the map at a later point.

Reply:
I don't have this information in the initialiser list. I have to parse and interpret the map file to fill it.
I could make it mutable, but this would not prevent internal functions to modify it, which is the case if they use the const reference.

Reply:
Ah, you wrote via function call. I will consider that for the next PR. I has been like that, and this PR is about minimal changes to get the new map file syntax in before the FW teams starts implementing it in RDL.

Reply:
I will create a ticket.